config-templates/ibm/ibm-object-storage-plugin/2.2/custom-parameters.json

```diff
@@ -40,15 +40,15 @@
         "supported-values": ["IBM", "AWS", "Wasabi"]
     },
     {
-        "description": "Enter COS storage class. For more info, refer to https://ibm.biz/cos-storage-classes",
-        "displayname": "COS storage class",
+        "description": "Enter Object storage region. For more info, refer to https://ibm.biz/cos-storage-classes",
+        "displayname": "Object Storage Region",
         "name": "cos-storageclass",
         "category": "config",
         "required": "true"
     },
     {
-        "description": "COS Endpoint. Required when 's3provider' is not set. Precedence is given to 's3provider' when both are set. For more information, refer to https://ibm.biz/cos-endpoints",
-        "displayname": "COS Endpoint",
+        "description": "Object storage service endpoint. Required when 'Object storage service provider' is not set. Preference is given to 'Object storage service provider' when both are set. For more information, refer to https://ibm.biz/cos-endpoints",
```

Comment:
`Object Storage Service Provider` we will have this string same as the display name of s3provider